app/components/Image/Image.js

```diff
@@ -7,47 +7,25 @@ import styles from './Image.css';
 type Props = {
   src: string,
   className?: string,
-  alt: string
+  alt: string,
+  style: Object
 };
 
-type State = {
-  loaded: boolean
-};
-
-class Image extends Component<Props, State> {
+class Image extends Component<Props> {
   static defaultProps = {
     alt: 'image'
   };
 
-  state = {
-    loaded: false
-  };
-
-  // https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html
-  _isMounted = false;
-
-  componentDidMount() {
-    this._isMounted = true;
-    const image = new global.Image();
-    image.src = this.props.src;
-    image.onload = () => this._isMounted && this.setState({ loaded: true });
-  }
-
-  componentWillUnmount() {
-    this._isMounted = false;
-  }
-
   render() {
-    const { src, className, alt, ...props } = this.props;
+    const { src, className, alt, style, ...props } = this.props;
+    const finalClass = cx(styles.image, className);
+    if (!src) return <div className={finalClass} style={style} />;
```

Comment:
Smooth!

We should try to switch almost all images to this

Reply:
we could also show a specific thing while the image itself loading by using `onLoad` on the `img`, but not sure if it's worth it